server_admin/topics/authentication/flows.adoc

```diff
@@ -342,29 +345,67 @@ claims= {
+will need to authenticate again with that level. This is useful for operations that require higher security in the application (e.g. send payment) and always require authentication
+with the specific level.
+
+WARNING: Note that parameters like `claims` or `acr_values` might be changed by the user in the URL when the login request is sent from the client to the {project_name} via the user's browser.
+This can be mitigated if the client uses PAR (Pushed authorization request), a request object, or other mechanisms, which prevents the user from rewriting the parameters in the URL.
+Hence after the authentication, clients are encouraged to check the ID Token to doublecheck that `acr` in the token corresponds to the expected level.
+
+In case that there is no explicit level requested by parameters, the {project_name} will require the authentication with the first LoA
+condition found in the authentication flow (Username/Password in the example above). When a user was already authenticated with that level
+and that level expired, the user is not required to re-authenticate, but `acr` in the token will have value 0. This is considered as authentication
+based solely on `long-lived browser cookie` as mentioned in the section 2 of OIDC Core 1.0 specification.
 
 NOTE: A conflict situation may arise when an admin specifies several flows, sets different LoA levels to each, and assigns the flows to different clients. However, the rule is always the same: if a user has a certain level, it needs only have that level to connect to a client. It's up to the admin to make sure that the LoA is coherent.
 
+*Example scenario*
+
+. Max Age is configured as 300 seconds for level 1 condition.
+. Login request is sent without requesting any acr. Level 1 will be used and user needs to authenticate with username and password. Token will have `acr=1`.
+. Another login request is sent after 100 seconds. User automatically authenticated due the SSO and token will return `acr=1`.
+. Another login request is sent after another 201 seconds (301 seconds since authentication in point 2). User automatically authenticated due the SSO, but token will return `acr=0` due the level 1 is considered expired.
+. Another login request is sent, but now it will explicitly request ACR of level 1 in the `claims` parameter. User will be asked to re-authenticate with username/password
+  and then `acr=1` will be returned in the token.
+
+*ACR claim in the token*
+
+ACR claim is added to the token by the `acr loa level` protocol mapper defined in the `acr` client scope. This client scope is the realm default client scope
+and hence will be added to all newly created clients in the realm.
+
+In case you do not want `acr` claim inside tokens or you need some custom logic for adding it, you can remove the client scope from your client.
+
+Note when login request initiates request with `claims` parameter requesting `acr` as `essential` claim, then {project_name} will always return
```

Comment:
Add "the" before "login"
Add "a" before "request"
add "the" before 'claims` paremeter